### DISCUSSION TITLE
Apple Podcast

### DISCUSSION THREAD

**Discussion by N1lzh** [ciderapp/Cider#1055] [Added Features]
I know that Cider is for Apple Music, but I think it would be a great enhancement if you include Apple Podcasts into the Cider app.

**Comment by coredev-uk** [comment]
It does have Apple Podcasts built-in.

**Reply by N1lzh** [reply]
Where? Can't find any Podcasts from Apple Podcasts when I search them.

**Reply by N1lzh** [reply]
Oh nvm, found it. I think it would be better if you could directly search for them tho.

**Comment by N1lzh** [comment]
> I know that Cider is for Apple Music, but I think it would be a great enhancement if you include Apple Podcasts into the Cider app.

I have to admit that I was wrong and Apple Podcast is already a feature. Although I think it would be better if you could directly search for them, Library -> Podcasts is kind of hidden.

**Reply by TechStudent10** [reply]
Agreed!
